api/app/Models/User.php

```diff
@@ -20,6 +22,30 @@
  * @property string $telephone
  * @property string $preferred_lang
  * @property array $roles
+ * @property string $job_looking_status
+ * @property string $current_province
+ * @property string $current_city
+ * @property boolean $looking_for_english
+ * @property boolean $looking_for_french
+ * @property boolean $looking_for_bilingual
+ * @property string $bilingual_evaluation
+ * @property string $comprehension_level
+ * @property string $written_level
+ * @property string $verbal_level
+ * @property string $estimated_language_ability
+ * @property string $is_gov_employee
+ * @property string $interested_in_later_or_secondment
+ * @property int $current_classification
```

Comment:
This is actually a uuid. Probably best represented by `string` here?

Reply:
Oops, I just noticed that we've described every id and foreign id as an int in these docblocks, that's not your fault 😆